src/site/content/en/blog/multi-screen-window-placement/index.md

```diff
@@ -340,7 +340,7 @@ fullscreen:
 
 ```js
 try {
-  const primaryScreen = (await getScreens()).filter((screen) => screen.primary)[0];
+  const primaryScreen = (await getScreens()).screens.filter((screen) => screen.isPrimary)[0];
   await document.body.requestFullscreen({ screen: primaryScreen });
 } catch (err) {
```

Comment:
Is the catch block really useful here? Your other sample above doesn't have it.

Reply:
It's just standard [Fullscreen API exception](https://developer.mozilla.org/en-US/docs/Web/API/Element/requestFullScreen#exceptions) handling here.